summary/src/main/resources/application.properties

```
@@ -0,0 +1,40 @@
+spring.application.name=summary
+
+server.port=8084
+management.endpoints.web.exposure.include=health,metrics,otlp
+management.tracing.sampling.probability: 1.0
+management.otlp.tracing.endpoint: http://localhost:4318/v1/traces
+management.otlp.metrics.export.url: http://localhost:4318/v1/metrics
+management.otlp.logging.endpoint: http://localhost:4318/v1/logs
+
+app.meiliMasterKey=CHANGE_ME_CHANGE_ME
+app.meiliHost=http://localhost:7700
+
+# Ollama configuration for embeddings
+app.ollamaBaseUrl=https://gpu.aet.cit.tum.de/ollama
+app.ollamaModel=mxbai-embed-large
+app.ollamaApiKey=${OLLAMA_API_KEY:}
+
+springdoc.swagger-ui.path=/docs
+springdoc.api-docs.path=/openapi
+
+# Database Configuration
+spring.datasource.url=jdbc:postgresql://db:5432/promptheus
+spring.datasource.username=postgres
+spring.datasource.password=postgres
+spring.datasource.driver-class-name=org.postgresql.Driver
+
+# JPA Configuration
+spring.jpa.hibernate.ddl-auto=validate
```

the other non-migrating services have this below here.

Likely this is a bad idea in the long run though.

```suggestion
spring.jpa.hibernate.ddl-auto=none
```